sdk/RELEASE.md

```diff
@@ -46,6 +46,7 @@
 * Load v1 and v2 component yaml into v2 ComponentSpec and convert v1 component
   spec to v2 component spec [\#6497](https://github.com/kubeflow/pipelines/issues/6497)
 * Format all Python files under SDK folder. [\#6501](https://github.com/kubeflow/pipelines/issues/6501)
+* Update upper bound on kubernetes constraint in Python SDK. [\#6532](https://github.com/kubeflow/pipelines/pull/6532)
```

Comment:
This can be written as `Depends on kubernetes>=8.0.0,<19`.
Also need to move this up to the current version section.

Reply:
Cool, fixed. Thanks for the feedback!